### DISCUSSION TITLE
[Feature Request] ATI 3D Rage emulation (Mach64-based)

### DISCUSSION THREAD

**Discussion by aquaboxs** [86Box/86Box#4722] [General]
I post this as General for feature request, so I would like to add ATI 3D Rage emulation ported over to 86Box,
Here's documentation for ATI Rage ([atirage.md](https://github.com/dingusdev/dingusppc/blob/master/zdocs/developers/atirage.md)) for DingusPPC, 

and Actually, Here I found this VGA BIOS from VGA Museum:
[3drage2dvd.zip](https://github.com/user-attachments/files/16635811/3drage2dvd.zip)
[3dragepropci.zip](https://github.com/user-attachments/files/16635812/3dragepropci.zip)

If you don't like my feature request, that's fine. but thanks in advance.

**Comment by TC1995** [comment]
Good luck about the 3D part of this.

**Comment by Torinde** [comment]
- https://github.com/86Box/86Box/pull/4926
